### ISSUE TITLE
Add unit tests for parsing data

### ISSUE THREAD

**Issue by barbeau** [CUTR-at-USF/gnss-ride-hailing-analyzer#6]
**Is your feature request related to a problem? Please describe.**
The project currently parses the GPSTest and Chicago TNC data, but there aren't any tests to verify implementation.

**Describe the solution you'd like**
We're already bundling sample data in the repo, so these can be moved to a "data" folder and out of main resources (because we really don't want to bundle them in a JAR release). This was unit tests could access them as well to parse them and confirm at least the first few records are parsed correctly.

**Describe alternatives you've considered**
N/A

**Comment by barbeau** [comment]
TODO for this:
* Create test source folder and test class (under `src/test/...`) - for example, see https://github.com/CUTR-at-USF/travel-behavior-analysis/blob/master/src/test/java/edu/usf/cutr/tba/test/TravelBehaviorUtilsTest.java.
* Move files to `src/test/resources/` - for example, see https://github.com/CUTR-at-USF/gtfs-realtime-validator/tree/master/gtfs-realtime-validator-lib/src/test/resources. These files can then be accessed from the unit tests
* In the unit tests instantiate the parser and pass in the file name in the test resources directory, and then get back the list of `Location`s. Then assert that each element in the returned `List` matches what is expected based on the file. Something like:

```
@Test
public void testGpsTestParser() {
GPSTestParser gpsTestParser = new GPSTestParser("gnss_log_xyz.txt");
List<Location> locations = gpsTestParser.parseFile();
Location l0 = locations.get(0);
assertEquals("gps", l0.getProvider());
assertEquals("28.34833", l0.getLatitude());
...
Location l1 = locations.get(1);
assertEquals("gps", l1.getProvider());
assertEquals("28.3253", l1.getLatitude());
...
}
```

I'd suggest using Kotlin for the tests as the code is usually more concise.

**Comment by BumbleFlash** [comment]
@barbeau Thank you for taking your time to explain this. New PR coming soon!

**Comment by BumbleFlash** [comment]
Issue resolved in #7
